examples/ERP/classify_P300_bi_illiteracy.py

```diff
@@ -0,0 +1,181 @@
+warnings.simplefilter(action="ignore", category=FutureWarning)
+warnings.simplefilter(action="ignore", category=RuntimeWarning)
+
+warnings.filterwarnings("ignore")
+
+set_log_level("info")
+
+##############################################################################
+# Initialization
+# ----------------
+#
+# 1) Create paradigm
+# 2) Load datasets
+
+paradigm = P300()
+
+datasets = [Cattan2019_VR_Il()]
+
+# reduce the number of subjects, the Quantum pipeline takes a lot of time
+# if executed on the entire dataset
+# n_subjects = 1
+# title = "Datasets: "
+# for dataset in datasets:
+#     title = title + " " + dataset.code
+#     dataset.subject_list = dataset.subject_list[0:n_subjects]
+
+##############################################################################
+# We have to do this because the classes are called 'Target' and 'NonTarget'
+# but the evaluation function uses a LabelEncoder, transforming them
+# to 0 and 1
+labels_dict = {"Target": 1, "NonTarget": 0}
+
+##############################################################################
+# Create Pipelines
+# ----------------
+#
+# Pipelines must be a dict of sklearn pipeline transformer.
+
+pipelines = {}
+
+pipelines["QuantumMDM-Dist"] = QuantumMDMWithRiemannianPipeline(
+    convex_metric="distance", quantum=True
+)
+
+pipelines["Voting QuantumMDM"] = QuantumMDMVotingClassifier(quantum=True)
+
+
+pipelines["QuantumSVM"] = QuantumClassifierWithDefaultRiemannianPipeline(
+    shots=1024,
+    nfilter=2,
+    dim_red=PCA(n_components=10),
+)
+
+pipelines["LDA"] = make_pipeline(
+    XdawnCovariances(
+        nfilter=4,
+        classes=[labels_dict["Target"]],
+        estimator="scm",
+        xdawn_estimator="lwf",
+    ),
+    TangentSpace(),
+    LDA(solver="lsqr", shrinkage="auto"),
+)
+
+pipelines["MDM"] = make_pipeline(
+    # applies XDawn and calculates the covariance matrix, output it matrices
+    XdawnCovariances(
+        nfilter=4,
+        classes=[labels_dict["Target"]],
+        estimator="scm",
+        xdawn_estimator="lwf",
+    ),
+    MDM(),
+)
+
+pipelines["Voting Q+C MDM"] = VotingClassifier(
+    [
+        ("Quantum MDM", pipelines["QuantumMDM-Dist"]),
+        ("MDM ", pipelines["MDM"]),
+    ],
+    voting="soft",
+)
+
+##############################################################################
+# Run evaluation
+# ----------------
+#
+# Compare the pipeline using a within session evaluation.
+
+evaluation = WithinSessionEvaluation(
+    paradigm=paradigm,
+    datasets=datasets,
+    overwrite=True,
+)
+
+results = evaluation.process(pipelines)
+
+print("Averaging the session performance:")
+print(results.groupby("pipeline").mean("score")[["score", "time"]])
+
+
+# ##############################################################################
+# # Plot Results
+# # ----------------
+# #
+# # Here we plot the results to compare two pipelines
+
+fig, ax = plt.subplots(facecolor="white", figsize=[8, 4])
+
+sns.stripplot(
```

Comment:
Ok. It will look like this but with a dotted line instead (result for the 1 subject in the dataset):

![Figure_1](https://github.com/pyRiemann/pyRiemann-qiskit/assets/6229031/34cee821-c39d-489c-ae49-243b7efc49d6)

There is still some problem with the QSVC, which we need to investigate, but do you see other pipelines to test?

Reply:
It's ok for me.

Reply:
Super, thanks :) We just need to solve the issue with QSVC. I see it is working correctly with the loans dataset, and I see I have the same problem if I skip the wrapper, and create the pipeline directly with sklearn. Maybe it is just wrongly parametred.